builtin/aws/ecs/platform.go

```diff
@@ -910,6 +918,78 @@ func (p *Platform) resourceServiceDestroy(
 	return nil
 }
 
+func (p *Platform) resourceAlbRedirectListenerCreate(
+	ctx context.Context,
+	sg terminal.StepGroup,
+	sess *session.Session,
+	log hclog.Logger,
+	alb *Resource_Alb,
+	externalIngressPort ExternalIngressPort,
+
+	state *Resource_Alb_RedirectListener,
+) error {
+
+	s := sg.Add("Initiating ALB Redirect Listener creation")
+	defer s.Abort()
+
+	var HTTPListener *elbv2.Listener
+
+	if externalIngressPort == 443 {
+		elbsrv := elbv2.New(sess)
+
+		listeners, err := elbsrv.DescribeListenersWithContext(ctx, &elbv2.DescribeListenersInput{
+			LoadBalancerArn: &alb.Arn,
+		})
+
+		if err != nil {
+			return status.Errorf(codes.Internal, "failed to create redirect listener: %s", err)
+		}
+
+		for _, listener := range listeners.Listeners {
+			if *listener.Protocol == "HTTP" {
+				HTTPListener = listener
+			}
+		}
+
+		if HTTPListener != nil && *HTTPListener.DefaultActions[0].Type == "forward" {
+			s.Update("HTTP listener use for forwarding. Please destroy previus deployment or try again deploy app for create redirect listener")
+		} else {
+			s.Update("Creating redirect listener")
+
+			out, err := elbsrv.CreateListenerWithContext(ctx, &elbv2.CreateListenerInput{
+				LoadBalancerArn: &alb.Arn,
+				Port:            aws.Int64(int64(80)),
+				Protocol:        aws.String("HTTP"),
+				DefaultActions: []*elbv2.Action{
+					{
+						RedirectConfig: &elbv2.RedirectActionConfig{
+							Host:       &p.config.ALB.FQDN,
```

Comment:
This can be nil - ALB config isn't required. Do we need a "Host" set for the redirect listener to work?